style.css

```diff
@@ -5,6 +5,14 @@
   font-family: Roboto, sans-serif;
 }
 
+.menu-items,
+.msg-icon,
+.deskH,
+#fname,
+#lname {
+  display: none;
+}
+
 body {
   display: flex;
```

Comment:
- [ ] There's a margin on the body of your page, it will be good to set the margin to 0 to avoid blank spaces at the top of your page.
- [ ] The body of the page should have a background image as shown in the design.